### DISCUSSION TITLE
Left/Right swipe examples in the wrong place

### DISCUSSION THREAD

**Discussion by LaurensVanDamme** [community/community#147787] [Mobile]
You are able to configure your swipe options in the app. They show an example of what each swipe option does. However, the example for the right swipe is shown with the left swipe option and vice versa.

![Screenshot_20241223_140732_GitHub.jpg](https://github.com/user-attachments/assets/ee554a63-b2ce-48ab-bd11-29df6e5056f2)

I have noticed this is only the case when you first open the link to go to your device's notification settings.

https://github.com/user-attachments/assets/19a481ae-04e0-446c-b575-be11ec0ef9cd

Edit: Added screen recording to clarify.

**Comment by aconsuegra** [comment]
Hi @LaurensVanDamme, thanks a lot for your feedback! Can you share the version of the app you are using?

**Comment by LaurensVanDamme** [comment]
Hi @aconsuegra the version is v1.190.0 (827)

**Reply by aconsuegra** [reply]
Thanks for your quick reply! I have been trying to reproduce the issue without luck so far. I'll create an issue for our engineer to investigate what could be happening

**Reply by LaurensVanDamme** [reply]
Luckily it's not a terrible bug 🙂 Maybe it is somewhat related to the OS? I'm using One UI 6.1 on Android 14. If I you need more information, let me know!
